galasa-managers-parent/galasa-managers-cicsts-parent/dev.galasa.cicsts.manager/src/main/java/dev/galasa/cicsts/CicsTerminal.java

```diff
@@ -38,4 +36,9 @@
      */
     boolean connectAtStartup() default true;
     
+    /**
+     * The CICS TS Manager will automatically log into the CICS TS region via CESL using the terminal with 
+     * the specified secure credentials when it connects
+     */
+    String loginCredentials() default "";
```

Comment:
thinking that this should be loginCredentialsTag rather than login credentials as the user will specify the tag that we will use to inquire on the the creds store and not the actual credentials

Reply:
Updated to `loginCredentialsTag` throughout